Update programmatic-filtering.md (#817)
telerik/maui-docs: controls/datagrid/filtering/programmatic-filtering.md

### Nested Property Text Filter Descriptor

The `NestedPropertyTextFilterDescriptor` is a descriptor which allows you to filter the nested properties.

It exposes the following properties:

* `PropertyName`&mdash;Gets or sets the name of the property that is used to retrieve the filter value.

> * `ItemPropertyGetter`&mdash;Sets a custom function that should implement the access to the nested property. The function gets the value accessed by the `PropertyName` as the input argument and it should return the nested property as output result. If `ItemPropertyGetter` is not set, the filter descriptor behaves the same as [TextFilterDescriptor](#text-filter-descriptor).

Reviewer: ```suggestion
* `ItemPropertyGetter`&mdash;Sets a custom function that implements the access to the nested property. The function gets the value accessed by the `PropertyName` as an input argument and returns the nested property as an output result. If `ItemPropertyGetter` is not set, the filter descriptor behaves the same as [TextFilterDescriptor](#text-filter-descriptor).
```

* `Operator`&mdash;Gets or sets the `TextOperator` value that defines how the `Value` member is compared with each value from the items source.
* `Value`&mdash;Gets or sets the value used in the comparisons. This is the right operand of the comparison.
* `IsCaseSensitive`&mdash;Gets or sets a value that determines whether the text comparisons will be case-sensitive. The default value is `True`.

```C#
var filterDescriptor = new NestedPropertyTextFilterDescriptor();
filterDescriptor.PropertyName = "MyProperty";
filterDescriptor.Operator = Telerik.Maui.Controls.Compatibility.Common.Data.TextOperator.EqualsTo;
filterDescriptor.Value = "Expected value";
filterDescriptor.ItemPropertyGetter = (originalPropertyValue) =>
{

>     // The "originalPropertyValue" is the object fetched by the PropertyName of the descriptor. In this case value of the MyProperty property.

Reviewer: ```suggestion
    // The 'originalPropertyValue' is the object fetched by the 'PropertyName' of the descriptor. In this case, this is the value of 'MyProperty'.
```

dataGrid.FilterDescriptors.Add(new DelegateFilterDescriptor() { Filter = new CustomFilter()});
```

> >important You could check our Programmatic Filtering example referring to the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}).

Reviewer: When referring to the SDKBrowser, always use the same sentence as a template. This makes the pages predictable and saves the developer time. You can adjust the sentence below so it fits this specific scenario:

"For a runnable example with the Button Getting Started scenario, see the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}) and go to Button > Getting Started."

Reviewer 2: Updated
